cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaStatusTest.java

```diff
@@ -998,6 +998,42 @@ public void testInitialStatusOnOldResource() throws ParseException {
         });
     }
 
+    @Test
+    public void testKafkaClusrerIdInStatus(VertxTestContext context) throws ParseException {
```

Comment:
I think this tests only one small aspect of this. I think this test would be much more useful, if you mock the Kafka Admin client and have that return a mocker ClusterID. And then in the reconciliation you actually should let the `kafkaGetClusterId` run as it is in the code. To have it tested as well.

Reply:
Agreed, I'll follow your pointer below, the test I include didn't test much at all.